nemo/collections/tts/modules/magpietts_inference/evaluate_generated_audio.py

```diff
@@ -32,23 +32,25 @@
 
 import nemo.collections.asr as nemo_asr
 from nemo.collections.asr.metrics.wer import word_error_rate_detail
+from nemo.collections.tts.modules.utmosv2 import UTMOSv2Calculator
 from nemo.utils import logging
 
 # Path to evalset config JSON
-EVALSET_CONFIG_PATH = Path(__file__).parent / 'evalset_config.json'
+# Get project root: from nemo/collections/tts/modules/magpietts_inference/ go up to NeMo/
+_project_root = Path(__file__).resolve().parents[5]
+EVALSET_CONFIG_PATH = _project_root / 'examples' / 'tts' / 'evalset_config.json'
```

Comment:
We should refactor this to work the same way as examples/tts/magpietts_inference.py